### ISSUE TITLE
Decreased circulating chenodeoxycholic acid concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7150]
**Preferred term label:**
Decreased circulating chenodeoxycholic acid concentration

**Synonyms**
Decreased circulating chenodiol concentration

**Definition (free text, please give PubMed ID)**
A reduced concentration of chenodeoxycholic acid in the blood circulation. 
Comment: Chenodeoxycholic acid (chenodiol) is a primary bile acid, synthesized in the liver and present in high concentrations in bile
**Parent term (use hpo.jax.org/app)**
Abnormal serum bile acid concentration HP:0030984

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
619481 
PMID: 28898457  2/2

**Your nano-attribution (ORCID)**
